packages/CodingStandard/src/Fixer/Property/ArrayPropertyDefaultValueFixer.php

```diff
@@ -70,8 +70,13 @@ private function isArrayPropertyDocComment(Token $token): bool
         return true;
     }
 
-    private function addDefaultValueForArrayProperty(Token $semicolonToken): void
+    private function addDefaultValueForArrayProperty(Tokens $tokens,int $semicolonPosition): void
     {
-        $semicolonToken->setContent(' = [];');
+        $tokens->insertAt($semicolonPosition, new Token(']'));
+        $tokens->insertAt($semicolonPosition, new Token('['));
+        $tokens->insertAt($semicolonPosition, new Token(' '));
```

Comment:
I don't mean to change PHP parser...

But have a helper developer friendly method like

```php
$token->replateContent(string $content);
```

That would do what you say for use. It would prevent such a long commenting and hours of beein gstuck.